### PR TITLE
Fw 4252 complete the list of parts of speech in fixtures

### PR DESCRIPTION
**Pull request by sahil97**

### CODE REVIEW

File: src/firstvoices/backend/fixtures/partsOfSpeech_initial.json [hunk 1]
```diff
@@ -261,5 +261,69 @@
...
+      "title": "Reflexive Verb",
+      "parent": [
+        "Verb"
+      ]
+    }
+  },
+  {
+    "model": "backend.partofspeech",
+    "fields": {
+      "title": "Entity",
```
Comment (sarahfirstvoices) [inline src/firstvoices/backend/fixtures/partsOfSpeech_initial.json:284]: From the wiki page it looks like this should be "Entity (Noun-like word)

File: src/firstvoices/backend/fixtures/partsOfSpeech_initial.json [hunk 2]
```diff
@@ -261,5 +261,69 @@
...
+    "model": "backend.partofspeech",
+    "fields": {
+      "title": "Word Piece/Affix",
+      "parent": null
+    }
+  },
+  {
+    "model": "backend.partofspeech",
+    "fields": {
+      "title": "Event/Activity",
```
Comment (sarahfirstvoices) [inline src/firstvoices/backend/fixtures/partsOfSpeech_initial.json:298]: From the wiki page it looks like this should be "Event/Activity (Verb-like word)"

Comment (sarahfirstvoices) [reply src/firstvoices/backend/fixtures/partsOfSpeech_initial.json:298]: Also on the wiki page this lists itself as a parent... I wonder if the parent is meant to be "Verb"?

**Review by sarahfirstvoices — CHANGES_REQUESTED** [review]
Comparing to the wiki page, a couple of the existing rows are also different. Could you please double check the title of "modifier verb" and the parent of "demonstrative"?

File: src/firstvoices/backend/fixtures/partsOfSpeech_initial.json (hunk 2 above)
Comment (sahil97) [reply src/firstvoices/backend/fixtures/partsOfSpeech_initial.json]: Oh yes, this does not have a parent. Have confirmed from @caforbes and have updated the wiki page as well.

**Review by NolanVH — APPROVED** [review]

**Review by sarahfirstvoices — APPROVED** [review]

**Review by caforbes — APPROVED** [review]
Looking good!

**Pull request merged**